app/views/admin/applications/show.blade.php

```diff
@@ -117,8 +117,14 @@
                 @endif
               </div>
             @endif
+            @else
```

Comment:
I mean I can switch it to that if you want. All I did was tack on the else to what was already there